application/controllers/artifacts_controller.py

```diff
@@ -68,6 +68,20 @@ def update(self, params, object_id):
         except NotFound:
             return {"error": "not found"}, 404
 
+    @use_args(artifacts_validator.update_many_args())
+    def update_many(self, params):
+        """ Logic for updating multiple artifacts at once """
+
+        for update_params in params["data"]:
+            object_id = update_params.pop("id")
+            try:
+                artifact = Artifact.find(object_id)
+                artifact.update(update_params)
```

Comment:
This can probably be improved later on to not do a database request for every single image but instead do batch processing. Fine for now though.